### PR TITLE
Provide a way to opt-in to EFF e-mail

### LINKED ISSUE #3807: Ask if you want to share your e-mail address with EFF

### PR DESCRIPTION
**Pull request by bmw**
Fixes #3807.

~~I'd appreciate a review on this, but don't merge it just yet. The EFF sign up works in that you'll get a confirmation e-mail, but it's not the confirmation e-mail we want. Let's wait until this is fully set up before people running from git start using it.~~

I created #4081 in response to `pylint` complaining about too many lines in `certbot/cli.py`.

### CODE REVIEW

**Comment by bmw** [comment]
@schoen, do you have spare cycles to take a look at this?

File: certbot/eff.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    :returns: True if we should subscribe the user, otherwise, False
+    :rtype: bool
+
+    """
+    prompt = (
+        'Would you be willing to share your email address with the '
+        "Electronic Frontier Foundation, a founding partner of the Let's "
+        'Encrypt project and the non-profit organization that develops '
+        "Cerbot? We'd like to send you email about our work to encrypt "
+        'the web and protect its users.')
```
Comment (SwartzCr) [inline certbot/eff.py]: did you come up with this wording - or did someone like peter or kurt have input?

Comment (bmw) [inline certbot/eff.py]: I took it verbatim from Richard.

File: certbot/eff.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    display = zope.component.getUtility(interfaces.IDisplay)
+    return display.yesno(prompt, default=False)
```
Comment (bmw) [inline certbot/eff.py]: The caller stores it: https://github.com/certbot/certbot/pull/4082/files#diff-a91ee912eb688499052bee8878de5308R28

**Pull request merged**